blog-service/2023-04-28-dashboard.md

```diff
@@ -0,0 +1,17 @@
+---
+title: April 28, 2023 (Dashboards)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - dashboards new
+  - dashboard migration
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're excited to announce the release of several new features in the **Dashboards (New)**:
```

Comment:
We should add how these features will benefit the user and where they can find them.